monasca_monitor/metrics_source.py

```diff
@@ -98,6 +98,7 @@ def _send_heartbeat_metric(client):
         try:
             client.metrics.create(**kwargs)
         except (keystoneauth_exceptions.connection.ConnectFailure,
+                keystoneauth_exceptions.connection.ConnectTimeout,
```

Comment:
Please can you post some notes in `https://github.com/stackhpc/monasca-monitor/issues/1` to describe the failure that this fixes?